### PR TITLE
1.1/384 more detailed payment history email feedback message

### PR DESCRIPTION
**Pull request by katjam**
@ob6160 You will cry - the user wasn't auth'd on behalf of that trader.

I simplified the tests too - but feel free to hack or merge.

### CODE REVIEW

File: tests/Unit/Controllers/Api/TraderControllerTest.php [hunk 1]
```diff
@@ -72,4 +73,59 @@ public function testShowVoucherHistoryCompilesListOfPaymentHistory()
...
+        // There should be some vouchers pended today.
+        $date = Carbon::now()->format('d-m-Y');
+        $this->actingAs($this->user, 'api')
+            ->json('POST', route('api.trader.voucher-history-email', 1), [
+                'submission_date' => null,
+            ])
+            ->assertStatus(202)
+            ->assertJson([
+                'message' => trans(
+                    'api.messages.email_voucher_history', [
```
Comment (katjam) [inline tests/Unit/Controllers/Api/TraderControllerTest.php]: hmmm passes but should this one be api.messages.email_voucher_history_date?

Comment (ob6160) [inline tests/Unit/Controllers/Api/TraderControllerTest.php]: It's because you're not passing the date in the initial post. It's still null :)

Comment (ob6160) [inline tests/Unit/Controllers/Api/TraderControllerTest.php]: And it should be _date

**Comment by katjam** [comment]
Oh yeah. Duh. Can you fix and merge old?

**Pull request merged**